.github/workflows/test-fips-e2e.yml

```
@@ -150,8 +154,8 @@ jobs:
       if: >
         !github.event.repository.private &&
         always()
-      uses: codecov/codecov-action@v4
+      uses: codecov/codecov-action@v5.4.2
```

```suggestion
      uses: codecov/codecov-action@v5.4.2
```
Let's pin the action with the commit SHA instead of the version number for security reasons.

---

Good idea, updated!